examples/syscall_steal.c

```diff
@@ -1,5 +1,5 @@
 /*
- * syscall.c
+ * syscall_steal.c
```

Comment:
> And rename syscall module name to syscall_steal because In Debian, the name syscall conflicts with the patch debian/patches/features/x86/x86-make-x32-syscall-support-conditional.patch

Could you please explain more details about the conflict?

Reply:
because of the [patch](https://lore.kernel.org/lkml/1415245982.3398.53.camel@decadent.org.uk/T/#u) in debian, there is a kernel parameter inside `/sys/module/syscall/`
I think the param prefix makes the name there and keeps another module with the same name from being loaded.
```
~/lkmpg/examples$ ll -R /sys/module/syscall/
/sys/module/syscall/:
total 0
drwxr-xr-x 2 root root    0 Dec 22 12:41 parameters
--w------- 1 root root 4096 Dec 21 14:39 uevent

/sys/module/syscall/parameters:
total 0
-r--r--r-- 1 root root 4096 Dec 22 12:41 x32
```
When I `insmod syscall.ko uid=xx`
```
~/lkmpg/examples$ sudo insmod syscall.ko uid=1000
insmod: ERROR: could not insert module syscall.ko: Invalid parameters
```
dmesg:
```
~/lkmpg/examples$ sudo dmesg | tail -n 1
[79500.176148] syscall: module is already loaded
```

Reply:
I see.
Then, you should add this information (including the patch source and mailing list) to the commit message.